generic/tests/cfg/iometer_windows.cfg

```diff
@@ -73,3 +73,36 @@
                     bg_mode = yes
                 - after_test:
                     bg_mode = no
+        - with_multi_scsi_disks:
+            only virtio_scsi
+            run_timeout = 3600
+            format_multi_disks = yes
+            images = "image1"
+            image_name_stg0 = images/storage0
+            image_name_stg1 = images/storage1
+            image_name_stg2 = images/storage2
+            image_name_stg3 = images/storage3
+            image_size_stg0 = 2G
+            image_size_stg1 = 2G
+            image_size_stg2 = 2G
+            image_size_stg3 = 2G
+            force_create_image_stg0 = yes
+            force_create_image_stg1 = yes
+            force_create_image_stg2 = yes
+            remove_image_stg0 = yes
+            remove_image_stg1 = yes
+            remove_image_stg2 = yes
+            Win7, Win8, Win8..1, Win10, Win2008r2:
+                icf_name = "iometer_multi_3disks.icf"
+                disk_indexes = "1 2 3"
+                disk_letters = "I J K"
+                disk_fstypes = "ntfs fat32 fat"
+                images += " stg0 stg1 stg2"
+            Win2012, Win2012..r2, Win2016, Win2019:
```

Comment:
can write to "Win2012, Win2016, Win2019:", Win2012 include Win2012-64 and Win2012-r2.

Reply:
ok, thanks, update it.